clients/python/client/osparc/_studies_api.py

```diff
@@ -36,3 +58,53 @@ def create_study_job(self, study_id, job_inputs, **kwargs):
     def clone_study(self, study_id, **kwargs):
         kwargs = {**kwargs, **ParentProjectInfo().model_dump(exclude_none=True)}
         return super().clone_study(study_id, **kwargs)
+
+    def studies(self) -> PaginationGenerator:
+        def pagination_method():
+            page_study = super(StudiesApi, self).list_studies(limit=20, offset=0)
+            assert isinstance(page_study, PageStudy)  # nosec
+            return page_study
+
+        return PaginationGenerator(
+            first_page_callback=pagination_method,
+            api_client=self.api_client,
+            base_url=self.api_client.configuration.host,
+            auth=self._auth,
+        )
+
+    def get_study_job_output_logfiles(self, study_id, job_id) -> Path:
+        return asyncio.run(
+            self.get_study_job_output_logfiles_async(study_id=study_id, job_id=job_id)
+        )
+
+    async def get_study_job_output_logfiles_async(self, study_id, job_id) -> Path:
+        """Download study logs. The log from each node will
+        appear as a file with the node's name in the directory"""
+        logs_map = super().get_study_job_output_logfile(study_id, job_id)
+        assert isinstance(logs_map, JobLogsMap)  # nosec
+        log_links = logs_map.log_links
+        assert log_links  # nosec
+        download_links = [link.download_link for link in log_links]
+        unique_node_names = ensure_unique_names([link.node_name for link in log_links])
+
+        tmp_dir = Path(TemporaryDirectory().name).resolve()
```

Comment:
1. What about passing a `download_dir: Path | None = None` in the inputs and default to a temporary
2. Use https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp  instead of `TemporaryDirectory()` since the latter is normally for directories where you want to have cleanup control